mixins/form.jsx

```diff
@@ -411,7 +417,13 @@ module.exports = {
         submitting: true
       });
       submitProps = form.buildProps(props);
-      this.submit("/api/signup", submitProps, this.signupSuccess, this.signupError);
+      this.submit(url, submitProps, success, error);
     }
+  },
+  signup: function(validate, props) {
+    this.doSignup("/api/signup", validate, props, this.signupSuccess, this.signupError);
```

Comment:
So my goal is so common paths are default. So that means never needing to specify that I want to use exact target. Exact target is the default under `/signup`, and not an optional flag to be turned on and off. Using it, you wouldn't even need to know that it is even called exact target, it's just the standard signup service we have chosen.

Mailchimp is the option, either via a variable or a different path (path is my preference).

If the path is shareable, which I explored, we would need to find a way to make this still secure: https://github.com/mozilla/donate.mozilla.org/blob/master/server.js#L93-L99

mailchimp returns different things than exact target. So we need to somehow have a variable set of response schema. I'm not super concerned with mailchimps response schema, but I don't want to remove exact targets schema.